modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-controller-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-autoscaling-ingress-controller_{context}"]
+= Autoscaling an Ingress Controller
+
+Automatically scale an Ingress Controller to dynamically meet routing performance or availablility requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+
+.Prequisites
+. You installed the OpenShift CLI (`oc`).
+. You installed Custom Metrics Autoscaler Operator.
+. You are in the `openshift-ingress-operator` namespace.
+
```

Comment:
We're missing bits, taken from: https://github.com/frobware/openshift-keda-examples
- Enable OpenShift monitoring for user-defined projects
- Create a Service Account
- Define a TriggerAuthentication with the Service Account's token
- Create a role for reading metrics from Thanos
- Add the role for reading metrics from Thanos to the Service Account

Reply:
Excellent, thanks Andrew. Updated